src/content/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro.mdx

```diff
@@ -0,0 +1,43 @@
+These experiences include service entities in Explorer with golden metrics and entity dashboards, a service overview dashboard, and recommended alerts. 
+
+## Get started
+To get started you need to send [Prometheus metric data](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic) to New Relic. 
+Each service might require additional prerequisites to enable and configure the metrics endpoint to be scrapped. You can review specific settings and requirements for each particular service in the [Prometheus integrations list](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro#list).
+
+## Find and use the data
+
+### Metrics
+
+Prometheus metrics are stored as [dimensional metrics](/docs/data-apis/understand-data/metric-data/query-metric-data-type/). You can query using [NRQL](/docs/telemetry-data-platform/get-data/apis/query-metric-data-type/) or use the Data Explorer to browse available metrics, facet, and filter by dimensions.
+
+### Entities
+Most of the integrations provide curated entities that are automatically identified and available in [New Relic Explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/) enabling all the platform capabilities such as golden metrics, entity dashboards, workloads, lookout, and service levels. 
+
+### Dashboards
+
+You can import service dashboards through [New Relic I/O](https://onenr.io/06vjAmnD3jP). Dashboards provide a high-level overview of the service performance and follow the recommended observability best-practices for each service. Once you imported, you can adapt and customize them to your specific requirements. 
+
+### Alerts
+
+You can import the recommended alert policies through [New Relic I/O](https://onenr.io/06vjAmnD3jP). Once imported, [alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) based on NRQL queries are available, but they won't be automatically enabled. Review the notification settings and adjust the condition as needed before turning them on.
+
+## Prometheus integrations list [#list]
+
+New Relic offers curated experiences for these services:
+
+* [Calico](/docs/infrastructure/prometheus-integrations/integrations-list/calico-integration)
+* [Cockroach Database](/docs/infrastructure/prometheus-integrations/integrations-list/cockroach-db-integration)
+* [CoreDNS](/docs/infrastructure/prometheus-integrations/integrations-list/core-dns-integration)
+* [Etcd](/docs/infrastructure/prometheus-integrations/integrations-list/etcd-integration)
+* [Ingress controller](/docs/infrastructure/prometheus-integrations/integrations-list/ingress-controller-integration)
```

Comment:
here as well I would point out that we are talking about nginx and not about a generic ingress controller

Reply:
Done. Now it's 'NGINX Ingress Controller'